lxd/device/disk.go

```diff
@@ -745,6 +745,28 @@ func (d *disk) startVM() (*deviceConfig.RunConfig, error) {
 			if d.config["pool"] != "" {
 				var revertFunc func()
 
+				// If the pool is ceph backed, don't mount it.
+				if d.pool.Driver().Info().Remote {
+					// Get the pool and volume names.
+					clusterName, userName := d.cephCreds()
+					// Configuration values containing :, @, or = can be escaped with a leading \ character.
```

Comment:
Can you move this to a separate function on this disk device to avoid duplicating the logic from https://github.com/lxc/lxd/pull/10308/commits/fe2d9857b685ea688a5cec3b299d9562b3f35b85#diff-30511c7ce3bcabd6a4ddc8624541cc2f5240a13b5e3b19aac8c787fdbd0e4720R721